spec/helpers_spec.cr

```diff
@@ -41,16 +41,6 @@ describe "Helper" do
     end
   end
 
-  describe "#produce_playlist_continuation" do
```

Comment:
Care to provide a comment on why these were deleted? Are they incorrect?

Reply:
The different ctoken (continuation token) functions tend to change regularly, because of youtube innertube API changes, and trying to keep the "reference" strings up to date takes quite some time (I spend 20 minutes trying to manually craft those tokens), for no real benefit, as the results are evident:
https://test.invidious.io/playlist?list=PLC8GlM8JtFN64eCdqwszCtFTZn2KJfDOn